mmv1/third_party/terraform/resources/resource_bigquery_table.go

```diff
@@ -586,6 +586,25 @@ func resourceBigQueryTable() *schema.Resource {
 								},
 							},
 						},
+						// AvroOptions: [Optional] Additional options if sourceFormat is set to AVRO.
+						"avro_options": {
+							Type:        schema.TypeList,
+							Optional:    true,
+							MaxItems:    1,
+							Description: `Additional options if source_format is set to "AVRO"`,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"use_avro_logical_types": {
+										Type:        schema.TypeBool,
+										Optional:    true,
+										Description: `If sourceFormat is set to "AVRO", indicates whether to interpret logical types as the corresponding BigQuery data type (for example, TIMESTAMP), instead of using the raw type (for example, INTEGER).`,
+										AtLeastOneOf: []string{
```

Comment:
We should actually make this required as there are no other options to be set in this block. If someone sets the `avro_options` block, we need a value set inside of it - and this is the only option right now.

Reply:
I was going to do the same but as in the API documentation it was mentioned optional, I did the same but in different manner, but adding Required is a better way. So I've made the suggested changes.